pkg/libvmi/vm.go

```diff
@@ -59,19 +59,6 @@ func NewVirtualMachine(vmi *v1.VirtualMachineInstance, opts ...VMOption) *v1.Vir
 	return vm
 }
 
-// Deprecated: WithRunning is deprecated.
-// While the running field itself is not deprecated, it is discouraged;
-// We try to steer users towards using RunStrategy which allow for greater variation of user states
-// To keep this option functional, we nil out RunStrategy to avoid conflict
-func WithRunning() VMOption {
```

Comment:
I guess we can remove the linter rule? https://github.com/kubevirt/kubevirt/commit/752a550cb501b61fa832a03c7b454fc68c1c6f0e#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9

Reply:
Thanks for the note.
done